services/watson-extension/src/templates/platform/integrations/integration_update_name_success.txt.jinja

```diff
@@ -0,0 +1 @@
+The name of your integration was successfully updated.
```

Comment:
For this response and the endpoint URL one, could we tell them the name it was updated to. i.e.

`Your integration's name was updated to -delta-`.
or
`I've updated your integration name to -delta-`.